installation/proc-install-rhdh-operator-admin.adoc

```diff
@@ -0,0 +1,53 @@
+
+For more information on Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope].
+====
+
+This section describes the installation procedure for installing {product} on your OpenShift Container Platform using the Operator.
+
+//You can use the {product} Operator in Red Hat OpenShift (OpenShift) to install {product}.
+
+.Prerequisites
+
+* You are logged in as an administrator on the OpenShift web console.
+* You have configured the appropriate roles and permissions within your project to create an application. See the link:https://docs.openshift.com/container-platform/4.14/applications/index.html[Red Hat OpenShift documentation on Building applications] for more details.
+* If installing a CI build from quay.io/rhdh, make sure that:
+** you have been granted permission to pull images from the `rhdh` organization on quay.io, and 
+** you have added your Quay token to the cluster global pull secret (link:https://docs.openshift.com/container-platform/4.14/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets[link]):
++
+[source]
+----
+oc get secret/pull-secret -n openshift-config --template='{{index .data ".dockerconfigjson" | base64decode}}' > /tmp/my-global-pull-secret.yaml
+
+oc registry login --registry="quay.io" --auth-basic="<user>:<token>" --to=/tmp/my-global-pull-secret.yaml
+
+oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/tmp/my-global-pull-secret.yaml
+
+rm -f /tmp/my-global-pull-secret.yaml
+----
+** you have added a link:https://gist.github.com/nickboldt/d54ba9d7264a5480d14eea6c3bd2ecdf[RHDH Operator Catalog Source]:
++
+[source]
+----
+curl -L https://gist.githubusercontent.com/nickboldt/d54ba9d7264a5480d14eea6c3bd2ecdf/raw/76a7f166a17f86ae8e766241b9d648c9821898bb/install-rhdh-next-build.sh | bash
+----
+
+.Procedure
+. In the *Administrator* perspective of the OpenShift web console, go to *Operators* → *OperatorHub*, search for Red Hat Developer Hub, and install the Red Hat Developer Hub Operator. For more info, see link:https://docs.openshift.com/container-platform/4.14/operators/admin/olm-adding-operators-to-cluster.html#olm-installing-from-operatorhub-using-web-console_olm-adding-operators-to-a-cluster[Installing from OperatorHub using the web console].
+. Create an OpenShift project to be used by your Backstage instance. For more information about creating a project in OpenShift, see the link:https://docs.openshift.com/container-platform/4.14/applications/projects/working-with-projects.html#creating-a-project-using-the-web-console_projects[Red Hat OpenShift documentation].
```

Comment:
just FYI we'll need to clarify that there are two channels available from which to subscribe to the operator. I can amend doc as part of https://issues.redhat.com/browse/RHIDP-1105